src/bluetooth-fw/nimble/advert.c

```diff
@@ -252,6 +257,25 @@ static void prv_handle_notification_rx_event(struct ble_gap_event *event) {
   }
 }
 
+#ifdef RECOVERY_FW
+// TODO: Implement UI to prompt user to accept/reject a repeat pairing request.
```

Comment:
Please file either a Linear or a GitHub (ideally) issue to track this, and cite it in this TODO comment.

Reply:
https://github.com/pebble-dev/pebble-firmware/issues/205